testsuite/tests/kuadrant/authorino/sharding/test_sharding.py

```diff
@@ -0,0 +1,75 @@
+"""Test for authorino sharding"""
+import pytest
+from weakget import weakget
+
+from testsuite.openshift.httpbin import Envoy
+from testsuite.openshift.objects.auth_config import AuthConfig
+from testsuite.openshift.objects.authorino import AuthorinoCR
+
+
+@pytest.fixture(scope="session")
+def authorino(openshift, blame, request, testconfig, label) -> AuthorinoCR:
+    """Custom deployed Authorino instance"""
+    if not testconfig["authorino"]["deploy"]:
+        return pytest.skip("Operator tests don't work with already deployed Authorino")
+
+    authorino = AuthorinoCR.create_instance(openshift,
```

Comment:
I would wait until #19 is merged and then used `authorino_parameters` fixture it includes. I would also migrate this test to operator dir as only there should we modify Authorino creation in any way